cmd/agent/entrypoint.go

```diff
@@ -249,6 +257,68 @@ func (ep *Entrypoint) getReporterMetrics() map[string]interface{} {
 	return map[string]interface{}{"enabled-features": ep.cfg.EnabledFeatures}
 }
 
+func (ep *Entrypoint) supportHandler(rw http.ResponseWriter, r *http.Request) {
+	ep.mut.Lock()
+	cfg := ep.cfg
+	ep.mut.Unlock()
+
+	if cfg.DisableSupportBundle {
+		rw.WriteHeader(http.StatusNotFound)
+		_, _ = rw.Write([]byte("404 - support bundle generation is disabled"))
```

Comment:
Since we are adding context, would be nice to return the command to enable it in the message.

Reply:
The support bundles are enabled by default, and disabled via the `-disable-support-bundles` flag. Do you think we should remind the user why it's disabled, or is it okay?

Reply:
I would still probably add the flag name so they if they find they want to turn it off they dont have to dig example `and support bundle generation can be enabled by removing the --disable-support-bundles` flag.

Reply:
Sounds good, done!